messaging/src/main/java/org/axonframework/eventhandling/AbstractEventBus.java

```diff
@@ -154,11 +154,12 @@ public void publish(@Nonnull List<? extends EventMessage<?>> events) {
 
     private List<EventMessage<?>> eventsQueue(UnitOfWork<?> unitOfWork) {
         return unitOfWork.getOrComputeResource(eventsKey, r -> {
-            Span span = spanFactory.createInternalSpan(() -> getClass().getSimpleName() + ".commit");
+            Span prepareCommitSpan = spanFactory.createInternalSpan(() -> getClass().getSimpleName() + ".prepareCommit");
```

Comment:
This looks like an enhancement i.o. a bug fix ;-)
Would be nice to be clear about stuff like this in the issue description.

Reply:
You are right. I could revert this, I think it's too noisy anyway

Reply:
![image](https://user-images.githubusercontent.com/3278878/220378077-9ccc1961-3c80-43aa-914e-ac0a0d52b153.png)
Reverted it, looks good